docs/application/native/guides/location-sensors/device-sensors.md

```diff
@@ -409,6 +409,24 @@ The following table lists the measurement data that the HRM LED green sensor pro
 | Timestamp                        | `unsigned long long` | -                                       | Microseconds |
 | values[0]: HRM green light value | `int`                | Min. value = 0<br> Max. value = 4194304 | -            |
 
+<a name="hrm_green_batch"></a>
+## Heart Rate Monitor LED Green batch Sensor
```

Comment:
## Heart Rate Monitor LED Green Batch Sensor

Suggestion @bbrto21  If you can, I would recommend using sentence case for all headings. 
Please refer to 
https://github.com/Samsung/tizen-docs/blob/master/styleguide/style.md#capitalization

@mijin85cho  what are your views on changing the headings to sentence case.